pandas/_libs/tslibs/conversion.pyx

```diff
@@ -782,3 +782,130 @@ cdef inline str _render_tstamp(int64_t val):
     """ Helper function to render exception messages"""
     from pandas._libs.tslib import Timestamp
     return str(Timestamp(val))
+
```

Comment:
is there a reason for the localization functions to exists here rather than in timezones.pyx?

Reply:
Really want to limit the points of contact with `pandas_datetimestruct` and the ensuing dependencies.